scripts/scripts.js

```diff
@@ -207,8 +207,8 @@ async function loadPage() {
   loadDelayed();
 }
 
-// TODO: Remove once chatbot is compatible with Helix domain
-loadScript('/scripts/mockxmlhttprequest.js');
+// Needed for chatbot to work in dev environments
+loadScript('/scripts/mockxmlhttprequest.js', { type: 'module' });
```

Comment:
@sdmcraft would it be possible to load this script as well in delayed phase ? Since chatbot script is also we are loading in delayed phase only ?

Reply:
done